### PR TITLE
Remove professional plus from in-product pricing

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Remove Professional Plus Support offering from in-product offering. This was only available in the Cloud plans modal.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-54779

#### Screenshots

**Before**
Screen Shot 2023-10-06 at 10 16 45 AM

**After**
Screen Shot 2023-10-06 at 10 35 30 AM

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/i18n/nl.json [hunk 1]
```diff
@@ -4385,16 +4385,13 @@
   "postypes.custom_open_pricing_modal_post_renderer.upgrade_professional": "Upgrade naar Professional",
   "postypes.custom_open_pricing_modal_post_renderer.userRequests": "{userRequests} vroegen toegang tot deze functie",
   "postypes.custom_open_pricing_modal_post_renderer.view_options": "Bekijk de upgrade-opties",
-  "pricing_modal.addons.247Coverage": "24x7 bereikbaar",
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/i18n/nl.json:4388]: We generally dont remove or edit other languages as they are automatically updated from Weblate

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/i18n/nl.json (hunk 1 above)
Comment (marianunez) [reply webapp/channels/src/i18n/nl.json:4388]: For edit, they will be updating the texts but does it hurt to delete ? Haven't had issues in the past

Comment (M-ZubairAhmed) [reply webapp/channels/src/i18n/nl.json:4388]: For either edit or deletion, it gets synchronised in weblate

Comment (marianunez) [reply webapp/channels/src/i18n/nl.json:4388]: Agreed, so if all are deleted then it doesn't hurt to have had them removed

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by nickmisasi — APPROVED** [review]

**Review by abhijit-singh — APPROVED** [review]
Looks good to me!

**Review by johndavidlugtu — APPROVED** [review]

**Pull request merged**